src/main/java/Jarvas.java

```diff
@@ -0,0 +1,76 @@
+import java.util.Scanner;
+public class Jarvas {
+    public static final String PARTITION_LINE = "____________________________________________________________";
+    public static void initialiseBot(){
+        System.out.println(" _____                                  ");
+        System.out.println("(___  )                                 ");
+        System.out.println("    | |   _ _  _ __  _   _    _ _   ___ ");
+        System.out.println(" _  | | /'_` )( '__)( ) ( ) /'_` )/',__)");
+        System.out.println("( )_| |( (_| || |   | \\_/ |( (_| |\\__, \\");
+        System.out.println("`\\___/'`\\__,_)(_)   `\\___/'`\\__,_)(____/");
+
+        System.out.println(PARTITION_LINE);
+        System.out.println("Hello! I'm Jarvas");
+        System.out.println("What can I do for you?");
+        System.out.println(PARTITION_LINE);
+    }
+    public static void terminateBot(){
+        System.out.println(PARTITION_LINE);
+        System.out.println("Bye, see you soon!");
+        System.out.println(PARTITION_LINE);
+    }
+    public static void runTime(){
```

Comment:
"Names representing methods must be verbs." 
Can change runTime() to a verb.

Reply:
Overlooked this thanks a lot!